classes/person.rb

```diff
@@ -31,13 +32,17 @@ def can_use_services?
   def correct_name
     @name
   end
-end
 
-person = Person.new(22, 'maximilianus')
-person.correct_name
-capitalized_person = CaptailizeDecorator.new(person)
-capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalizedPerson)
-capitalized_trimmed_person.correct_name
+  # add_rental
+  def add_rental(rental)
+    @rentals << rental unless @rentals.include?(rental)
+  end
```

Comment:
- [ ] The method for adding rentals should have parameters book and date. Check this [link](
https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) to a previous example.
`def add_rental(book, date)`

- [ ] The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
`Rental.new(date, book, self)`